edgePy/data_import/mongodb/mongo_import.py

```diff
@@ -128,35 +127,35 @@ def get_data_from_mongo(
         sample_names = set()
         sample_category = {}
         for result in cursor:
-            print(result)
+            log.info(result)
             sample_names.add(result["sample_name"])
             sample_category[result["sample_name"]] = (
                 result[self.search_key] if self.search_key else result["sample_name"]
             )
-        print(f"get data.... for sample_names {list(sample_names)}")
+        log.info(f"get data.... for sample_names {list(sample_names)}")
 
         query = {"sample_name": {"$in": list(sample_names)}}
         if self.gene_list:
-            print(self.gene_list)
+            log.info(self.gene_list)
             query["gene"] = {"$in": list(self.gene_list)}
         cursor = self.mongo_reader.find_as_cursor(
             database=database, collection="RNASeq", query=query, projection={"_id": 0}
         )
 
         # make it a list of lists
-        print(f"Importing data from mongo ({self.mongo_host})....")
+        log.info(f"Importing data from mongo ({self.mongo_host})....")
         dataset: Dict[Hashable, Dict[Hashable, Optional[int]]] = {}
         gene_list = set()
         sample_list = set()
         count = 0
         for result in cursor:
```

Comment:
Why aren't we using `enumerate()` here? Dedicating a variable to count items in an iterable is an anti-pattern.

Reply:
I'm glad you pointed this out in general. I need to change those f types for the logging. Should be using `%`